app/models/ability.rb

```diff
@@ -2,11 +2,11 @@ class Ability
   include CanCan::Ability
 
   def initialize(user)
-    cannot :read, :all
-    return unless user.present?
-
-    can :read, :all, user_id: user.id
-    can :create, Inventory, user_id: user.id
-    can :destroy, Inventory, user_id: user.id
+    user ||= User.new # guest user (not logged in)
+    if user.admin?
+      can :manage, :all
+    else
+      can :read, :all
+    end
```

Comment:
We don't have an admin in this project so please leave the original ability file in the dev branch.